frontend/integration-tests/tests/networking.scenario.ts

```diff
@@ -0,0 +1,45 @@
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import { execSync } from 'child_process';
+import * as networkingView from '../views/networking.view';
+import { $$} from 'protractor';
+import * as secretsView from '../views/secrets.view';
+import * as crudView from '../views/crud.view';
+
+
+describe('Interacting with route creation forms', () => {
+  beforeAll(async() => {
+    execSync(`kubectl create -f integration-tests/data/caddy-docker.json -n ${testName}`);
```

Comment:
I don't know if this is the correct way to add some test files before running the suites, correct and point me the right path please.